README.md

```diff
@@ -337,6 +337,64 @@ are done with the migration.
 
   Modify this column if desired.
 
+### `partitions` ###
+
+- `schema` (type `name`): schema of the table with the partition
+
+- `table_name` (type `name`): name of the related table
+
+- `partition_name` (type `name`): name of the partition
+
+- `orig_name` (type `name`): name of the partition in the remote source
+
+- `type` (type `text`): one of the supported partitioning methods `LIST`,
+  `RANGE` or `HASH`
+
+- `expression` (type `text`): partitioning expression used to define a table's
+  partitioning
+
+- `position` (type `integer`): defines the order of the table partitions (must
+  start at 1)
```

Comment:
Do we really need that? Partitions don't have an order.
I know you use it to determine the number of hash partitions, but that could just as well be determined by the row count of the result set.
You could `MOVE` to the end and
```
GET DIAGNOSTICS integer_var = ROW_COUNT;
```

Reply:
`position` column has been deleted as discussed in #12 

See https://github.com/fljdin/db_migrator/commit/209a2e12b15bf9082ac4c8e28cdb9226829e9883